ftpsync/targets.py

```diff
@@ -52,7 +52,10 @@ def prompt_for_password(url, user=None):
     if user is None:
         default_user = getpass.getuser()
         while user is None:
-            user = raw_input("Enter username for ftp://%s [%s]: " % (url, default_user))
+            try: # python 2
+                user = raw_input("Enter username for ftp://%s [%s]: " % (url, default_user))
+            except: # python 3
```

Comment:
Generally we should catch the concrete exception type only (plain 'except' would also catch Ctrl-C, I guess)